go.mod

```diff
@@ -105,3 +104,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/kiegroup/container-builder v0.0.0-20221028160425-7921fe1891f1 => github.com/davidesalerno/kogito-builder v0.0.0-20221209101055-e9ac23a06d2d
```

Comment:
do not forget to remove before merging

Reply:
Updated after merging the other PR on the container-builder side!